File: java/Peripherals.java

public class Peripherals extends BaseIO {
    private String deviceType;
    private String deviceName;
    private String location;
    private String brand;

    public Peripherals(String lastDevice, String lastOutput, String deviceType, String deviceName, String location, String brand) {
        super(lastDevice, lastOutput);
        this.deviceType = deviceType;
        this.deviceName = deviceName;
        this.location = location;
        this.brand = brand;
    }

    public String getDeviceType() {
        return deviceType;
    }

    public String getDeviceName() {
        return deviceName;
    }

    public String getLocation() {
        return location;
    }

    public String getBrand() {
        return brand;
    }

    public void setDeviceType(String deviceType) {
        this.deviceType = deviceType;
    }

    public void setDeviceName(String deviceName) {
        this.deviceName = deviceName;
    }

    public void setLocation(String location) {
        this.location = location;
    }

    public void setBrand(String brand) {
        this.brand = brand;
    }

    public String info() {
        return "Peripherals: " + deviceType + " " + deviceName + " " + location + " " + brand;
    }
}

File: Python/Peripherals.py
from Base_io import BaseIO


class Peripherals(BaseIO):
    def __init__(self, last_device="", last_output="", device_type="", device_name="", location="", brand=""):
        super().__init__(last_device, last_output)
        self.device_type = device_type
        self.device_name = device_name
        self.location = location
        self.brand = brand

    def get_device_type(self):
        return self.device_type

    def get_device_name(self):
        return self.device_name

    def get_location(self):
        return self.location

    def get_brand(self):
        return self.brand

    def set_device_type(self, device_type):
        self.device_type = device_type

    def set_device_name(self, device_name):
        self.device_name = device_name

    def set_location(self, location):
        self.location = location

    def set_brand(self, brand):
        self.brand = brand

    def info(self):
        return f"Peripherals: {self.device_type} {self.device_name} {self.location} {self.brand}"